haskell-org-committee.markdown

```diff
@@ -14,15 +14,21 @@ Haskell.org is a 501(c)(3) non-profit organization that oversees the Haskell.org
   * [Discourse][discourse]
   * [Hoogle], a type-aware Haskell search engine
 
-Haskell.org also organizes Haskell's participation in  the [Google Summer of Code program][summer].
+Its purpose is to promote educational and scientific progress relating to the Haskell programming language and related technologies.
```

Comment:
I think this "scientific progress" could go, but I wanted to leave it as-is on the first commit, and I'm not sure what to replace it with (and then once its gone, the sentence needs more work).

Reply:
I'm pretty sure this is just quoting our mission statement from our 501(c)(3) documents—as I understand it, this is stock language educational nonprofits use to describe why they can be a nonprofit.

Reply:
OK, I guess the statement should remain as-is?